client/src/components/basic/task-instructions/components/Statistics/statistics.module.scss

```diff
@@ -0,0 +1,24 @@
+:local(.info) {
```

Comment:
file or folder name should follow [JS quality criteria](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md#naming)